src/Database/LSMTree/Internal/RunBuilder.hs

```diff
@@ -60,49 +60,49 @@ type RefCount = IORef Int
 --
 -- __Not suitable for concurrent construction from multiple threads!__
 --
-data MRun fhandle = MRun {
+data RunBuilder fhandle = RunBuilder {
       -- | The reference count for the LSM run. This counts the
       -- number of references from LSM handles to this run. When
       -- this drops to zero the open files will be closed.
-      lsmMRunRefCount   :: !RefCount
+      lsmRunBuilderRefCount   :: !RefCount
 
       -- | The file system paths for all the files used by the run.
-    , lsmMRunFsPaths    :: !RunFsPaths
+    , lsmRunBuilderFsPaths    :: !RunFsPaths
 
       -- | The run accumulator. This is the representation used for the
       -- morally pure subset of the run cnstruction functionality. In
       -- particular it contains the (mutable) index, bloom filter and buffered
       -- pending output for the key\/ops file.
-    , lsmMRunAcc        :: !(RunAcc ST.RealWorld)
+    , lsmRunBuilderAcc        :: !(RunAcc ST.RealWorld)
 
       -- | The byte offset within the blob file for the next blob to be written.
-    , lsmMRunBlobOffset :: !(IORef Word64)
+    , lsmRunBuilderBlobOffset :: !(IORef Word64)
 
       -- | The (write mode) file handles.
-    , lsmMRunHandles    :: {-# UNPACK #-} !(ForRunFiles (ChecksumHandle fhandle))
+    , lsmRunBuilderHandles    :: {-# UNPACK #-} !(ForRunFiles (ChecksumHandle fhandle))
```

Comment:
I think I'd like to remove the `lsm` field prefixes. They're only here and on `Run` itself and to me they just add noise without much benefit.
Do you think it makes sense to remove them? If so, you could do it for `RunBuilder` here (since you touch them already) and I could make a PR for `Run` a little later?

Reply:
Sounds good